js/script.js

```diff
@@ -3,300 +3,107 @@ let outfits = [{
     imgSrc: 'images/1.jpg',
     rank: 0,
     rejectedCounter: 0,
-    items: [' silk blouse', ' beige trouser', ' beltblazer', ' loafer', '  sunglasses', '  tote bag'],
+    items: [0,1,2,3,4,5],
+    tags: ['sport', ' classic']
+  },
+  {
+    outfitId: 2,
+    imgSrc: 'images/2.jpg',
+    rank: 0,
+    rejectedCounter: 0,
+    items: [1,2,3],
     tags: ['sport', ' classic']
   },
   {
     outfitId: 3,
     imgSrc: 'images/3.jpg',
     rank: 0,
     rejectedCounter: 0,
-    items: [' sweatshirt', ' leather trousers', ' heels', ' chain necklace'],
+    items: [1,2,4,5],
```

Comment:
this is ok to use kinda ids in here. But I would suggest to use enumeration. Which would be something like what was there before, but this would be unique identifiers for the items.

```js
  ...
  items: ['heels', 'sweatShirt', 'chainNeckless'],
  ...
```
and then the obects(links) will be
```js
let links = {
  heels: {
    name: 'slip dress',
    src: 'https://www.asos.com/search/?q=white+sneakers' 
  },
  ...
}
```